src/web/demo.rkt

```diff
@@ -317,6 +318,18 @@
      (redirect-to (add-prefix (format "~a.~a/graph.html" job-id *herbie-commit*)) see-other))
    (url main)))
 
+(define (get-timeline req job-id)
+  (match (get-results-for job-id)
+    [job-result
+     (response 202 #"Job in progress" (current-seconds) #"text/plain"
+               (list (header #"X-Job-Count" (string->bytes/utf-8 (~a (job-count)))))
+               (λ (out) (write-json (job-result-timeline job-result) out)))]
+    [#f
+     (response/full 201 #"Job complete" (current-seconds) #"text/plain"
+                    (list (header #"Location" (string->bytes/utf-8 (add-prefix (format "~a.~a/graph.html" job-id *herbie-commit*))))
+                          (header #"X-Job-Count" (string->bytes/utf-8 (~a (job-count)))))
+                    '())]))
+
```

Comment:
I'm confused here. If `get-results-for` returns `#f` that means the job is complete? If so, why redirect to `graph.html` instead of `timeline.json`?